homeassistant/components/zha/api.py

```diff
@@ -336,12 +353,10 @@ async def websocket_get_groupable_devices(
                     "endpoint_id": ep_id,
                     "entities": [
                         {
-                            "name": zha_gateway.ha_entity_registry.async_get(
-                                entity_ref.reference_id
-                            ).name,
-                            "original_name": zha_gateway.ha_entity_registry.async_get(
-                                entity_ref.reference_id
-                            ).original_name,
+                            "name": _get_entity_name(zha_gateway, entity_ref),
+                            "original_name": _get_entity_original_name(
+                                zha_gateway, entity_ref
+                            ),
```

Comment:
`async_get` can return a nullable, and `mypy` complains about the availability of `name`/`original_name`
I think this is better than adding a `#type: ignore`, even though it's more verbose